slide-deck.css

```diff
@@ -3,19 +3,59 @@ slide-deck {
   container: slide-deck / inline-size;
   display: grid;
 
-  &[slide-view=grid] {
+  &[slide-view=grid],
+  &[slide-view=script] {
     --slide-ratio: 16/9;
-    --target-outline: medium dotted;
     --target-margin: var(--gap);
-    gap: var(--gap);
-    grid-template-columns: repeat(auto-fill, minmax(min(50ch, 100%), 1fr));
+    --target-outline: medium dotted;
+    --target-outline-offset: calc(var(--gap) * 0.5);
+    gap: var(--row-gap, var(--gap)) var(--column-gap, var(--gap));
     padding: var(--gap);
   }
 
-  &[slide-view=list] {
+  &[slide-view=grid] {
+    --slide-note-margin-block: var(--gap);
+    grid-template-columns: repeat(auto-fill, minmax(min(50ch, 100%), 1fr));
+  }
+
+  &[slide-view=solo] {
     --slide-height: 100svh;
   }
 
+  &[slide-view=script] {
+    --column-gap: calc(1.25em + 2vw);
```

Comment:
To me this looks like more space than is needed? I think a faint boarder would be more useful than space? This feels like a view where context is more useful than whitespace.

Reply:
@mirisuzanne how much space are you seeing? I see like, less than pinky width on active slides? 

Official measurement device.

Reply:
@stacyk Oh oh oh, it's the `5em` row gap that's real large for me. Sorry. I think that could be more like a hairline and something closer to the column gap.

Reply:
I started playing with a hairline (it was a little finicky because of the slide item with container and with only canvas are different sizes), but then I wondered if was more than a layout thing that I should apply in a custom theme instead of the default styles in this tool. But I can play with it more in this tool to see if I can get it just right.